tox.ini

```diff
@@ -23,12 +23,19 @@ commands =
     twine check dist/*
 
 [testenv:release]
-basepython = python3
+skip_install = True
```

Comment:
The only way we can properly dog-food our own release is by using it to release twine. By installing the current master, we gain the benefit of one last ditch chance of catching a critical bug.

Reply:
I think you're conflating two concerns: dog-fooding and integration tests. The tool is properly dog-fooded because it uses twine to upload releases. For integration tests, I agree - cutting one's own release against the about-to-be-released version would provide additional assurance, especially until some more sophisticated integration tests can be added.

My remaining reluctance remains because I'd like this project to vary as little as possible from other projects. Each deviation from the standard practice (in this case, jaraco/skeleton) adds a cognitive and maintenance burden. My instinct is that the value gained from having a shared, consistent process is greater than the value gained by relying on the current version of twine to release the current version of twine; that it's highly unlikely that after sufficient testing, this change will catch an intrinsic error.

I'll apply the suggestion in the spirit of cooperation.

Reply:
`jaraco/skeleton` is just your own standard practice. Please don't make it sound like it's a global standard of excellence